### PR TITLE
Buttons Page

### PR DESCRIPTION
**Pull request by radityadd**
Unify buttons with dynamic ripple 👍 tulung direview pak.

@afifkhaidir-tkpd

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -86,12 +86,14 @@ const webpackConfig = {
...
         test: /\.html$/,
-        use: [{
-          loader: 'html-loader',
-          options: {
-            attrs: ['img:src', 'video:src'],
+        use: [
+          {
+            loader: 'html-loader',
+            options: {
+              attrs: [':src'],
```
Comment (afifkhaidir-tkpd) [inline webpack.config.js]: Jangan pake `[':src']` dit, kalau terlalu global kena `<script src="...">` juga

File: src/sass/_ripple.scss [hunk 2]
```diff
@@ -0,0 +1,104 @@
+@mixin keyframes($animation-name) {
```
Comment (afifkhaidir-tkpd) [inline src/sass/_ripple.scss]: Ini kayaknya mixins keyframes bisa dipake buat global, mending tambah ke `_mixins.scss` aja

File: src/js/index.js [hunk 3]
```diff
@@ -3,6 +3,77 @@
...
+    spanEl.style.width = `${size}px`;
+    spanEl.style.height = `${size}px`;
+
+    offsetX = self.ownerDocument.defaultView.pageXOffset;
+    offsetY = self.ownerDocument.defaultView.pageYOffset;
+
+    rippleX = parseInt(ev.pageX - (self.getBoundingClientRect().left + offsetX), 10) - (size / 2);
+    rippleY = parseInt(ev.pageY - (self.getBoundingClientRect().top + offsetY), 10) - (size / 2);
+
+    console.log(self.ownerDocument.defaultView);
```
Comment (afifkhaidir-tkpd) [inline src/js/index.js]: console.log dihapus aja dit. Btw kalau mau jalanin ESLint, tinggal pake command 
```bash
$ ./node_modules/bin/eslint {nama_file}
````

Comment (radityadd) [reply src/js/index.js]: https://github.com/afifkhaidir-tkpd/webpack-static-boilerplate/pull/2/commits/4957ed51f48359d53ff667a0ba321650867948f5

File: src/sass/_ripple.scss (hunk 2 above)
Comment (radityadd) [reply src/sass/_ripple.scss]: https://github.com/afifkhaidir-tkpd/webpack-static-boilerplate/pull/2/commits/5243f7c1250236fc5d31c8275f5228fc2b0fcaa6

https://github.com/afifkhaidir-tkpd/webpack-static-boilerplate/pull/2/commits/7692eaa8a3b9bd154cc3136def7229f689c87958

File: webpack.config.js (hunk 1 above)
Comment (radityadd) [reply webpack.config.js]: https://github.com/afifkhaidir-tkpd/webpack-static-boilerplate/pull/2/commits/57ca8732e0c04d119cf2f2e6a57015c0b1c11e7b

**Comment by radityadd** [comment]
@afifkhaidir-tkpd syudah ya pak

**Pull request merged**